### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by syepes** [syepes/VSphere2Metrics#1]
Add to following items:
- [ ] Readme
- [ ] Usage examples
- [ ] Installation / Setup
- [ ] Grafana Dashboards

**Comment by Papalloon** [comment]
Yes, please.

**Comment by syepes** [comment]
@Papalloon Yes need some spare time, but right now its pretty simple to get it working:
- Download the built [jar](https://github.com/syepes/VSphere2Metrics/releases/latest)
- Encrypt the user password `java -jar VSphere2Metrics.jar -pwd PasswordToEncrypt`
- `config.groovy`: Set your desired metrics destination and vcs.urls, vcs.user and use the previously encrypted password in vcs.pwd
- Now you just need to start the jar...
- Have fun! :-)
